platform/core/src/classes/StudyPrefetcher.js

```diff
@@ -204,6 +204,7 @@ export class StudyPrefetcher {
       'SeriesInstanceUID',
       image.imageId
     );
+    if (!study) return null;
```

Comment:
I'm wondering what getSeriesByUID and getInstanceByUID return if not found? maybe it already returns null?